lib/puppetserver/ca/cli.rb

```diff
@@ -28,8 +28,8 @@ class Cli
       INIT_ACTIONS = {
         'import'   => Action::Import,
         'setup'    => Action::Setup,
-        'enable' => Action::Enable,
-        'migrate' => Action::Migrate,
+        'enable'   => Action::Enable,
+        'migrate'  => Action::Migrate,
```

Comment:
I thought we decided we weren't going to do this kind of alignment? My IDE does it automatically, but I usually put it back, so I just wanna be clear on where we've landed with that.

Reply:
I don't think it's idiomatic to do it in clojure, I do think it's idiomatic in Ruby. I don't know that I'm the best at following that though.

Reply:
ahh, okay. I don't really care, just wasn't sure what we had agreed on.